setup.py

```diff
@@ -110,7 +110,13 @@ def is_requirement(line):
     url='https://github.com/edx/event-bus-kafka',
     packages=[
         'edx_event_bus_kafka',
+        'edx_event_bus_kafka.consumer'
     ],
+    entry_points={
+        "lms.djangoapp": [
```

Comment:
[question] My understanding of `entry_points` is still vague, but it seems odd to me that there would be an edxapp reference here. Wouldn't we want some standardized name that refers to event-bus implementations instead?

Reply:
Good call. Unfortunately, we don't really have a good way to do this yet (the documentation for plugins says `For a plugin that will appear in a single IDA, you could create constants like [ProjectType found in edx-platform](https://github.com/edx/edx-platform/blob/dbe40dae1a8b50fea0954e85f76ebf244129186e/openedx/core/djangoapps/plugins/constants.py#L14-L22). If the plugin is for many IDAs, we need to add a capability to this library with a global constant.` Given that I think we're on track to make this a library instead of plugin, I vote we ignore this peculiarity for now.

Reply:
From conversation with Robert, we can drop all the plugin-related stuff -- it was only needed for edx-arch-experiments, and we'll be using a different mechanism to hook up the implementation in this new library. ([openedx-events looks at an optional Django setting](https://github.com/openedx/edx-platform/issues/30682) and loads some module or class name from the implementation that references the producer and consumer code.)

Reply:
I vote we drop that in the next PR since the whole point of this one was to move over the old code exactly so we can more clearly see what we are changing when we abstract it out.